samples/instances/suspend-resume/resume.js

```diff
@@ -16,8 +16,8 @@
  * Resumes a suspended Google Compute Engine instance (with unencrypted disks).
  *
  * @param {string} projectId - Project ID or project number of the Cloud project you want to use.
- * @param {string} zone - Name of the zone to create the instance in. For example: "us-west3-b"
- * @param {string} instanceName - Name of the new virtual machine (VM) instance.
+ * @param {string} zone - Name of the zone where the instance belongs to. For example: "us-west3-b"
```

Comment:
grammar: "zone where the instance belongs" (best) or "zone the instance belongs to" (OK)